backends/advanced/src/advanced_omi_backend/controllers/system_controller.py

```
@@ -272,171 +247,100 @@ async def get_speaker_service_status():
-            return {
-                "message": "Memory configuration updated and reloaded successfully",
-                "config_path": config_path,
-                "backup_created": os.path.exists(backup_path),
-                "status": "success"
-            }
-        else:
-            return JSONResponse(
-                status_code=500, content={"error": "Configuration saved but reload failed"}
-            )
-        
+            raise ValueError(f"Invalid YAML syntax: {str(e)}")
```

Ruff (0.14.8)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)